### ISSUE TITLE
Creating a wallet is blocked by lack of accessibility using voice over

### ISSUE THREAD

**Issue by Cal-L** [MetaMask/metamask-mobile#4886]
### **Description**

Lack of accessibility that blocks onboarding - Need to be able to tab into the SRP options in the last step of onboarding using voice over.

### **Technical Details**

- Figure out how to make the individual SRP words accessibility using voice over

### **Acceptance Criteria**

- Should be able to tab into the SRP words in the last step of the onboarding flow using voice over

### **References**

-  Mobile accessibility audit notes - https://www.notion.so/8-18-22-Mobile-Accessibility-Audit-70161b16ca894ea8a183b6cf6d03b29e

**Comment by DostoevskysPiano** [comment]
@Cal-L Is this still something we need to do? Or can we close it? 

Unable to access notes.
